src/components/bulkConcepts/bulkConceptList.jsx

```diff
@@ -1,42 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import {
-  Form,
-  FormGroup,
```

Comment:
Is the deletion of this file required as part of the ticket?

Reply:
Yes, the Text Area  which is the only thing the component renders is now on line 165 of src/components/bulkConcepts/addBulkConcepts.jsx, where the selected search results are displayed.